syntaxhighlighter3/styles/shCore.css

```diff
@@ -7,7 +7,7 @@
  *
  * @version
  * 3.0.83 (July 02 2010)
- * 
+ *
```

Comment:
Just for documentation purposes:

I thought it was a vendor file, but it seems there are some other changes that were applied to this file already, like this one: https://github.com/Automattic/syntaxhighlighter/commit/9fc4a5893e7af79f4f6831e33211f4e625589614

So probably it's not a big deal to change this file. Also, since that lib is not being maintained anymore, probably we won't have new versions to update this file. :)